index.js

```diff
@@ -121,6 +125,26 @@ function doTransform(k) {
       );
     } else if (ts.isImportClause(n) && n.isTypeOnly) {
       return ts.createImportClause(n.name, n.namedBindings);
+    } else if (ts.isTypeReferenceNode(n) && n.typeName.escapedText === "Omit") {
+      const symbol = checker.getSymbolAtLocation(n.typeName);
+
+      if (
+        symbol !== undefined &&
+        symbol.declarations.length > 0 &&
+        symbol.declarations[0]
+          .getSourceFile()
+          .fileName.includes("node_modules/typescript/lib/lib")
+      ) {
+        const child = n.getChildAt(2);
```

Comment:
can you use properties of `TypeReference` instead of `Node.getChildAt`? It would make it easier to read; for example it would make clearer that only two different children are used, and that they are type arguments.

Reply:
Sure, I updated the code. Thanks for the tip.